packages/itwin/measure-tools/src/api/FormatterUtils.ts

```diff
@@ -55,14 +55,26 @@ export class FormatterUtils {
     return FormatterUtils.formatCoordinatesWithSpec(point, coordSpec);
   }
 
-  public static formatCoordinatesImmediate(point: Point3d): string {
-    const coordSpec =
-      IModelApp.quantityFormatter.findFormatterSpecByQuantityType(
-        QuantityType.Coordinate
-      );
-    if (undefined === coordSpec) return "";
-
-    return FormatterUtils.formatCoordinatesWithSpec(point, coordSpec);
+  public static formatCoordinatesImmediate(point: Point3d, coordSpec?: FormatterSpec): string {
+    let result: string;
+    if (!coordSpec) {
+      coordSpec =
+        IModelApp.quantityFormatter.findFormatterSpecByQuantityType(
+          QuantityType.Coordinate
+        );
+      if (undefined === coordSpec) return "";
+      result = FormatterUtils.formatCoordinatesWithSpec(point, coordSpec);
+    } else {
+      // eslint-disable-next-line @itwin/no-internal
+      const oldFormatTraits = coordSpec.format.formatTraits;
```

Comment:
@aruniverse these apis are marked as beta, but linter flags warnings for them complaining about the no internal rule. Is the rule a bug or should I disable the no internal rule line by line like this (or disable across file)

Reply:
I believe this is because of this config in package's `eslint.config.js`:
```
      "@itwin/no-internal": [
        "warn",
        {
          tag: [
            "internal",
            "alpha",
            "beta",
          ],
        },
      ],
```

Reply:
thanks, I've omitted alpha and beta from the warning

Reply:
id recommend leaving alpha in, need to be careful using those apis

Reply:
alright, i added alpha back